ddl/table.go

```diff
@@ -156,11 +167,50 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 	return ver, errors.Trace(err)
 }
 
+func createTableWithForeignKeys(d *ddlCtx, t *meta.Meta, job *model.Job, tbInfo *model.TableInfo, fkCheck bool) (ver int64, err error) {
+	switch job.SchemaState {
+	case model.StateNone:
+		err = checkTableForeignKeyValidInOwner(d, job, tbInfo, fkCheck)
```

Comment:
wenjun concurrent ddl check.

Reply:
Since currently only has 1 general DDL worker, so doesn't  need to consider concurrent `create table` ddl.